### PR TITLE
WFP-1416 display previously managed on choose practitioner page correctly

### PR DESCRIPTION
**Pull request by markreesmoj**

### CODE REVIEW

File: integration_tests/mockApis/allocationCase.ts [hunk 1]
```diff
@@ -85,78 +85,4 @@ export default {
       },
     })
   },
```
Comment (markreesmoj) [inline integration_tests/mockApis/allocationCase.ts:87]: This one is used on other pages so I have left it for now.

File: integration_tests/mockApis/allocations.ts [hunk 2]
```diff
@@ -27,7 +27,6 @@ export default {
         sentenceDate: '2021-09-01',
         initialAppointment: null,
         status: 'Previously managed',
-        previousConvictionEndDate: '2019-12-13',
```
Comment (markreesmoj) [inline integration_tests/mockApis/allocations.ts:30]: never read

File: server/models/CaseForChoosePractitioner.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import OffenderManager from './OffenderManager'
+
+export default interface CaseForChoosePractitioner {
+  name: string
+  crn: string
+  tier: string
+  status: string
+  offenderManager: OffenderManager
```
Comment (sp-watson) [inline server/models/CaseForChoosePractitioner.ts:8]: We could use `?` to show that this value is optional

**Review by sp-watson — DISMISSED** [review]
Nice!

**Review by sp-watson — APPROVED** [review]

**Review by carlov20 — APPROVED** [review]

**Pull request merged**